src/countBy.ts

```diff
@@ -62,6 +61,8 @@ export namespace countBy {
   /**
    * Counts how many values of the collection pass the specified predicate.
    *
+   * @param array - The input data.
```

Comment:
countBy is going away, can you disable the rule at the top of the file instead of fixing it to prevent merge conflicts with #571